torch/_overrides.py

```diff
@@ -651,8 +666,203 @@ def get_testing_overrides():
+        Tensor.numpy: lambda self: -1,
+        Tensor.permute: lambda self, dim1, dim2: -1,
+        Tensor.pin_memory: lambda self: -1,
+        Tensor.put_: lambda self, indices, tensor, accumulate=False: -1,
+        Tensor.qscheme: lambda self: -1,
+        Tensor.random_: lambda self, from_=0, to=None, *, generator=None: -1,
+        Tensor.record_stream: lambda self, stream: -1,
+        Tensor.refine_names: lambda self: -1,
+        Tensor.register_hook: lambda self, hook: -1,
+        Tensor.rename: lambda self, name: -1,
+        Tensor.repeat: lambda self, size1, size2: -1,
+        Tensor.requires_grad_: lambda self, requires_grad=True: -1,
+        Tensor.reshape_as: lambda self, other: -1,
+        Tensor.resize: lambda self, size1, size2: -1,
+        Tensor.resize_as: lambda self, other: -1,
+        Tensor.retain_grad: lambda self: -1,
+        Tensor.set_: lambda self, source=None, storage_offset=0, size=None, stride=None: -1,
+        Tensor.share_memory_: lambda self: -1,
+        Tensor.short: lambda self, memory_format=torch.preserve_format: -1,
+        Tensor.size: lambda self: -1,
+        Tensor.sparse_dim: lambda self: -1,
+        Tensor.sparse_mask: lambda self: -1,
+        Tensor.sparse_resize_: lambda self, size1, size2: -1,
+        Tensor.sparse_resize_and_clear_: lambda self, size1, size2: -1,
+        Tensor.sspaddmm: lambda self, mat1, mat2, beta=1, alpha=1, out=None: -1,
+        Tensor.storage: lambda self: -1,
+        Tensor.storage_offset: lambda self: -1,
+        Tensor.storage_type: lambda self: -1,
+        Tensor.stride: lambda self, dim: -1,
+        Tensor.sum_to_size: lambda self, size1, size2: -1,
+        Tensor.to: lambda self, dtype, non_blocking=False, copy=False, memory_format=torch.preserve_format: -1,
+        Tensor.to_dense: lambda self: -1,
+        Tensor.to_sparse: lambda self, sparseDims: -1,
+        Tensor.tolist: lambda self: -1,
+        Tensor.to_mkldnn: lambda self: -1,
+        Tensor.type_as: lambda self, other: -1,
+        Tensor.unflatten: lambda self, dim, namedshape: -1,
+        Tensor.uniform_: lambda self, from_=0, to=1: -1,
+        Tensor.values: lambda self: -1,
+        Tensor.view: lambda self, shape1, shape2: -1,
+        Tensor.view_as: lambda self, other: -1,
+        Tensor.zero_: lambda self: -1,
     }
 
+    ret2 = {}
+    ignored = get_ignored_functions()
+
+    for k, v in ret.items():
+        names = [
+            k.__name__,
+            k.__name__ + "_",
+            "__" + k.__name__ + "__",
+            "__i" + k.__name__ + "__",
+            "__r" + k.__name__ + "__"
+        ]
+
+        if k.__name__.startswith("bitwise_"):
+            subname = k.__name__[len("bitwise_"):]
+            names.extend([
+                subname,
+                subname + "_",
+                "__" + subname + "__",
+                "__i" + subname + "__",
+                "__r" + subname + "__"
```

Comment:
Twice the same set of five names being generated here - extract into a utility function? Again, please add a comment - not all readers will be familiar with the Python dunder methods and/or Pytorch `name_` variant.

Reply:
No longer the same five names. :)